dockers/docker-lldp/start.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
+CFGGEN_PARAMS=" \
+    -d \
+    -t /usr/share/sonic/templates/lldpd.conf.j2 \
+    -y /etc/sonic/sonic_version.yml \
+    -t /usr/share/sonic/templates/lldpdSysDescr.conf.j2 \
+"
```

Comment:
do you want the lldod.conf an lldpdSysDescr.conf to be int a single file? if not, you could rewrite as:
```
CFGGEN_PARAMS=" \
￼    -d \
￼    -t /usr/share/sonic/templates/lldpd.conf.j2,<file path> \
￼    -y /etc/sonic/sonic_version.yml \
￼    -t /usr/share/sonic/templates/lldpdSysDescr.conf.j2,<file path> \
￼"
```